lib/scss/components/event-card/stanford_events.event-card.scss

```diff
@@ -1,5 +1,9 @@
 @charset 'UTF-8';
 @import '../../stanford_events.config';
+@import 'event-stacked-date--card';
+@import 'event-spacing-explore-more--card';
+@import 'event-card--heading-link';
+@import 'event-card--item-type-link';
 
 .stanford-events {
   &.cards {
```

Comment:
Heres another occurance of targeting views when its on the event card pattern. We should have these things isolated so that the card has its own styles and the views adds styles to change the layout of multiple cards, but not the card itself.